user/models.py

```diff
@@ -1,5 +1,16 @@
 from django.contrib.auth.models import AbstractUser
+from django.core.exceptions import ValidationError
 
 
 class User(AbstractUser):
-    pass
+    @staticmethod
+    def validate_password(password: str, error_to_show):
+        if len(password) < 5:
+            raise error_to_show({
+                "password": "Password must be more than or equal to 5 symbols."
+            })
+
+    def clean(self):
+        User.validate_password(
+            self.password, ValidationError
+        )
```

Comment:
In the `clean` method, you are directly passing `self.password` to the `validate_password` method. However, `self.password` in Django's user model is a hashed value, not the actual password. Therefore, the length check will not work correctly. You might need to redesign your approach to password validation. Consider using Django's built-in validators or customizing the `set_password` method instead.